packages/ra-ui-materialui/src/input/FileInputPreview.tsx

```diff
@@ -42,6 +50,7 @@ FileInputPreview.propTypes = {
     className: PropTypes.string,
     file: PropTypes.object,
     onRemove: PropTypes.func.isRequired,
+    removeIcon: PropTypes.element,
```

Comment:
Careful with the prop's casing. I'd suggest using lower camel case everywhere (i.e. `removeIcon` instead of `RemoveIcon`), here and in `FileInput.tsx`

Reply:
I agree that the post you linked does recommend to use upper-casing for this case, and this is probably a good practice indeed.

However, in the case of react-admin, there are already many places in the code where component props are lower-case, hence for the sake of code consistency I'd rather have the `removeIcon` prop lower-case too.

You can find examples with [the `Layout.tsx` component](https://github.com/marmelab/react-admin/blob/master/packages/ra-ui-materialui/src/layout/Layout.tsx).

[Code](https://github.com/marmelab/react-admin/blob/master/packages/ra-ui-materialui/src/layout/Layout.tsx#L22-L32):

```tsx
    const {
        appBar: AppBar = DefaultAppBar,
        children,
        className,
        dashboard,
        error: errorComponent,
        menu: Menu = DefaultMenu,
        sidebar: Sidebar = DefaultSidebar,
        title,
        ...rest
    } = props;
```

[Documented usage](https://marmelab.com/react-admin/Layout.html#usage):

```tsx
import { Layout } from 'react-admin';

import { MyAppBar } from './MyAppBar';

export const MyLayout = props => <Layout {...props} appBar={MyAppBar} />;
```

Reply:
Ok, thanks for the clarification! I changed the prop to camel case everywhere.